platform/viewer/public/config/default.js

```diff
@@ -19,6 +19,19 @@ window.config = {
       },
     ],
   },
+  routesValidationMode: 'log', //[log, silent, fail]
```

Comment:
👀 Is it worth having this separate from the global `process.env.NODE_ENV` or `process.env.DEBUG` values?

Reply:
yeah..